app/controllers/api/calls_controller.rb

```diff
@@ -16,7 +17,8 @@ def create
 
   def call_params
     params.require(:call)
-      .permit(:member_phone_number, :target_id)
+      .permit(:member_phone_number, :target_id, :target_title, :target_name,
+              :target_phone, :target_country_code, :target_phone_extension, :checksum)
```

Comment:
I was thinking it would be much simpler to pass the data in plain text and then validate that it passes the checksum, the same way that AKIDs work. The advantage of this to me is that way we can see the numbers clearly in our logs, rather than needing to decrypt them to see that info about the intended target.